### PR TITLE
[WIP] The Hunt for 1.0.0

### PR DESCRIPTION
**Pull request by zenangst**
## How to contribute to this PR.

1. Make a branch from `develop` and make pull request pointing to `develop`
## Mockups
*Coming soon*

### CODE REVIEW

**Comment by 3lvis** [comment]
This should be split in different issues/tasks and they should be grouped under a Milestone.

**Comment by zenangst** [comment]
@3lvis I made a milestone and moved the tasks. What do you think?

**Comment by 3lvis** [comment]
Awesome!!! Is this good now?

**Comment by zenangst** [comment]
Should we have this PR going until `1.0.0` is ready or should we do them as separate pull request? I like the idea of having a general discussion going but it might be overkill. What do you guys think?

**Comment by 3lvis** [comment]
Sounds like an overkill to me. Could become hard to follow and mixed conversations happening at once.

Discussion should get place on their specific PR, if you want to start a new "topic" you'll create a new GitHub issue just to clarify your question or idea.

**Comment by zenangst** [comment]
Yah, it might be better to have more contextual discussions. I guess this is ready to go for now.

File: Pod/Demo/AppDelegate.swift [hunk 1]
```diff
@@ -19,8 +21,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     UINavigationBar.appearance().barTintColor = UIColor(fromHex:"FF5703")
     UINavigationBar.appearance().barStyle = UIBarStyle.BlackTranslucent
 
-    let leftButton = UIBarButtonItem(title: "Previous Page", style: .Plain, target: tutorialController, action: "previousPage")
-    let rightButton = UIBarButtonItem(title: "Next Page", style: .Plain, target: tutorialController, action: "nextPage")
+    let leftButton = UIBarButtonItem(title: "Previous page", style: .Plain, target: tutorialController, action: "previous")
+    let rightButton = UIBarButtonItem(title: "Next page", style: .Plain, target: tutorialController, action: "next")
```
Comment (3lvis) [inline Pod/Demo/AppDelegate.swift]: :O nice! `target: tutorialController`

Comment (zenangst) [inline Pod/Demo/AppDelegate.swift]: :grin:

File: Source/TutorialModel.swift [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    imageView.autoresizingMask = .FlexibleLeftMargin | .FlexibleRightMargin
+
+    return imageView
+    }()
+
+  lazy private(set) var titleLabel: UILabel = {
+    let label = UILabel(frame: CGRectNull)
+
+    label.autoresizingMask = .FlexibleLeftMargin | .FlexibleRightMargin
+    label.font = UIFont(name: "DIN Alternate", size: 48.0)
```
Comment (3lvis) [inline Source/TutorialModel.swift]: Kind of weird that we supply a font for the default things.

We could easily drop font and textColor here and make it a requirement, same as Form.

File: Source/TutorialModel.swift [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    label.textColor = UIColor(fromHex: "234583")
+
+    return label
+    }()
+
+  lazy private(set) var textLabel: UILabel = {
+    let label = UILabel(frame: CGRectNull)
+
+    label.autoresizingMask = .FlexibleLeftMargin | .FlexibleRightMargin
+    label.font = UIFont(name: "DIN Alternate", size: 32.0)
```
Comment (3lvis) [inline Source/TutorialModel.swift]: Same here with the font

File: Source/TutorialModel.swift (hunk 2 above)
Comment (zenangst) [inline Source/TutorialModel.swift]: Agreed! :)

**Comment by 3lvis** [comment]
Will make a PR

**Comment by zenangst** [comment]
Great!
